File: FosServer/src/main/java/com/example/fostracker/models/Address.java
package com.example.fostracker.models;

public class Address {
    String street;
    String area;
    String city;
    String state;
    String pincode;
    String country;

    public Address(String street, String area, String city, String state, String pincode, String country) {
        this.street = street;
        this.area = area;
        this.city = city;
        this.state = state;
        this.pincode = pincode;
        this.country = country;
    }

    public String getStreet() {
        return street;
    }

    public void setStreet(String street) {
        this.street = street;
    }

    public String getArea() {
        return area;
    }

    public void setArea(String area) {
        this.area = area;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getPincode() {
        return pincode;
    }

    public void setPincode(String pincode) {
        this.pincode = pincode;
    }
    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }
}

File: frontend/gpay_merchant/lib/address.dart
class Address {
  String street;
  String area;
  String city;
  String state;
  String pincode;
  String country;

  Address(this.street, this.area, this.city, this.state, this.pincode,
      this.country);

  Map<String, dynamic> toJson() => {
        'street': street,
        'area': area,
        'city': city,
        'state': state,
        'pincode': pincode,
        'country': country
      };
}
